tc-build.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -ex
+
+source $(dirname $0)/tc-vars.sh
+
+build_gpu=no
+build_arm=no
+
+if [ "$1" = "--gpu" ]; then
```

Comment:
Those scripts are not really intended to have any use outside of taskcluster, so I prefer to keep them as simple as possible for now. A while loop, for now, seems too complex for the simple usecase here :)